docs/Documentation/06-publishing.md

```diff
@@ -1,21 +1,19 @@
 # Publishing in Studio
 
-Whenever changes are made to your API descriptions or Markdown documentation, whether that is done in Stoplight Studio or anywhere else, you want to get those changes reflected in Documentation, Mock Servers, Explorer, the Design Library, etc.
+When changes are made to your API descriptions or Markdown documentation, whether that is done in Stoplight Studio or anywhere else, you want to get those changes reflected in Documentation, Mock Servers, Explorer, etc.
 
-In Studio v1.x publishing was done by clicking the "Publish" button, but with the launch of Stoplight Platform publishing can be [automated using Webhooks, or Continuous Integration](https://meta.stoplight.io/docs/platform/2.-workspaces/g.automating-publishing.md). 
+The publishing method you use depends on your project type:
 
-## Published Documentation URL
+* **Git Projects**: [Connect your Git repository](../02-git-credentials.md) and make updates automatically in Stoplight whenever changes are made within Studio or another editor, such as VS code. See [Publish Git Projects](https://meta.stoplight.io/docs/platform/ZG9jOjM0MjE2NTA5-git-projects).
+* **Stoplight Projects**: Publish changes as soon as they are made in Studio Web. This method is not available in Studio Desktop. See [Publish Stoplight Projects](https://meta.stoplight.io/docs/platform/ZG9jOjM0MjIzMjI0-stoplight-projects).
+* **CLI Projects**: Use the Stoplight CLI to publish local projects, projects that do not use Git, or to facilitate continuous integration. You can also use the Stoplight CLI to automate publishing directly from any Git provider. We simplify this for GitHub by providing a GitHub Workflow. See Publish with the Stoplight CLI. See [Publish with the Stoplight CLI](https://meta.stoplight.io/docs/platform/ZG9jOjQ1NTQxMw-stoplight-cli).
 
-If you click the Publish button in Studio Desktop v1.x, your documentation will go here:
+Note that nodeJS version 12 or greater is required to run the Stoplight CLI. You can verify your version using the command: `node --version`
 
-```
-https://stoplight.io/p/docs/gh/{org}/{project}
-```
-
-If you have created a workspace on [Stoplight Platform](https://stoplight.io/welcome) then you no longer need to click that button (it'll be gone when you upgrade to Studio Desktop v2.0 anyway) and the documentation will instead be over here.
-
-```
-https://{workspace}.stoplight.io/docs/{project}
-```
-
-_If you need any help migrating from the old documentation to the new documentation, please [contact support](mailto:support@stoplight.io)._
+> **Learn More**
+> 
+> [Publish Overview](https://meta.stoplight.io/docs/platform/ZG9jOjQ1NTQxNA-publishing)
+> 
+> [Add Projects in Studio Web](https://meta.stoplight.io/docs/platform/ZG9jOjE4ODEyMw-add-projects)
+> 
+>[Add Projects in Studio Desktop](../Basics/01-working-with-projects.md)
```

Comment:
The use of a block quote to format this seems a little strange to me, instead of a second-level heading and section containing some bullet points.